packages/visitors/visitors.20170127/opam

```diff
@@ -21,4 +21,4 @@ depends: [
   "ppx_tools"
   "ppx_deriving" {>= "4.0"}
 ]
-available: [ ocaml-version >= "4.03" ]
+available: [ ocaml-version >= "4.03" & ocaml-version < 4.05 ]
```

Comment:
I think you are simply missing the quotes around the version.

Reply:
(cf. @camelus's feedback)

Reply:
Yes, this just needs the quotes and it should be good to merge. Thanks for fixing up the older revisions!